README.Rmd

```diff
@@ -174,7 +195,7 @@ server <- shinyServer(function(input, output) {
 shinyApp(ui = ui(), server = server)
 ```
 
-How to contribute?  {.darkslide}
+How to contribute?
```

Comment:
i suggest to make this more user friendly :) I can take care of that and will add a PR.

Reply:
would be great, I think styleguide is needed for sure, anything else?